buildengine/build_go_test.go

```diff
@@ -300,7 +300,8 @@ func TestGoModVersion(t *testing.T) {
 }
 
 func TestGeneratedTypeRegistry(t *testing.T) {
-	if !testing.Short() {
+	t.Skip("FLAKY")
```

Comment:
Was this flaking in CI? I tried running it locally with increasingly absurdly high values in `go test -count=N` and couldn't reproduce a failure

Reply:
We should probably make an issue to fix this and unskip the test, so we don't forget.

Reply:
Yeah it was failing very often. [This](https://github.com/TBD54566975/ftl/issues/1580) is the issue.

Reply:
awesome, thanks!